client/benches/tps/lib.rs

```diff
@@ -77,7 +77,7 @@ impl Config {
         }
         // END
         let elapsed_secs = timer.elapsed().as_secs_f64();
-        thread::sleep(core::time::Duration::from_secs(2));
+        thread::sleep(core::time::Duration::from_secs(4));
```

Comment:
```suggestion
        thread::sleep(core::time::Duration::from_secs(self.block_apply_lag_secs));
```
and add the corresponding to the configuration